chipsec/hal/igd.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+#CHIPSEC: Platform Security Assessment Framework
+#Copyright (c) 2010-2017, Intel Corporation
+# 
+#This program is free software; you can redistribute it and/or
+#modify it under the terms of the GNU General Public License
+#as published by the Free Software Foundation; Version 2.
+#
+#This program is distributed in the hope that it will be useful,
+#but WITHOUT ANY WARRANTY; without even the implied warranty of
+#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#GNU General Public License for more details.
+#
+#You should have received a copy of the GNU General Public License
+#along with this program; if not, write to the Free Software
+#Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+#Contact information:
+#chipsec@intel.com
+#
+
+# -------------------------------------------------------------------------------
+#
+# CHIPSEC: Platform Hardware Security Assessment Framework
+#
+# -------------------------------------------------------------------------------
+
+"""
+Working with Intel processor Integrated Graphics Device (IGD)
+
+usage:
+    >>> gfx_aperture_dma_read(0x80000000, 0x100)

```

Comment:
Should fix usage documenting both read and write cases via gfx_aperture_dma_read_write